CHANGELOG.rst

```diff
@@ -54,6 +54,7 @@ Fixed
 
 Added
 ~~~~~
+* core.st2.CronTimer : A scheduled task cannot be executed at a given time. Add a new cron function. set the milestone #5779
```

Comment:
The changelog record looks irrelevant to the PR change here.
Could you update it?

Reply:
@chain312 Still need to update the Changelog here